content/en/docs/concepts/storage/volumes.md

```diff
@@ -1115,6 +1115,25 @@ persistent volume:
   secret is required. If the secret object contains more than one secret, all
   secrets are passed.
 
+#### CSI raw block volume support
+
+{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
+
+Starting with version 1.11, CSI introduced support for raw block volumes.  This work
+relies on the raw block volume feature that was introduced in previous versions of 
+Kubernetes.  This feature will make it possible for vendors with external CSI drivers to 
+implement raw block volumes support in Kubernetes workload.
+
+CSI block volume support is feature-gated and turned off by default.  To run CSI with
+block volume support enabled, add the following feature gate flag:
```

Comment:
It would be very helpful for operators to know which components needs this new feature gate. It is pretty clear that kubelet will need it and a safe way to do this is to add it to all control plane components. However, clearly specify which components need this would be useful.

Also, please consider add the `CSIBlockVolume` to the feature gates reference file : https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md

Reply:
@tengqm thanks for your input. I will make the necessary update to indicate that the Kubelet needs this. I will add the feature to the feture-gate doc.